classroom.rb

```diff
@@ -0,0 +1,15 @@
+require './student'
+
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ]  I suggest that you only allow the read-option for the student-classroom association. This is because no writing is done by the classroom class on the student class, it only checks/reads their status. `attr_reader :students`